crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -24,6 +24,9 @@ def status
       end
 
       def checks
+        upgrade_status = ::Crowbar::UpgradeStatus.new
+        upgrade_status.start_step if upgrade_status.current_step == "upgrade_prechecks"
```

Comment:
I think comment should be here explaining, that the check for current_step is there so we can allow executing this (and only this) step in any time